.mailmap

```diff
@@ -21,3 +21,4 @@ Jonathan Woollett-Light <jcawl@amazon.co.uk> <jonthanwoollettlight@gmail.com>
 karthik nedunchezhiyan <karthik.n@zohocorp.com>
 Babis Chalios <bchalios@amazon.es> <babis.chalios@gmail.com>
 Pablo Barbáchano <pablob@amazon.com>
+Nikita Kalyazin <kalyazin@amazon.co.uk> <kalyazin@amazon.com>
```

Comment:
There are also 2 entries for Jonathan on lines 19 and 20.

Reply:
Yep, that's intended. What that does is prefer the `.co.uk` to the other 2 variants (one has a typo).